examples/bigcommerce-catalyst/next.config.js

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+const withMakeswift = require('@makeswift/runtime/next/plugin')();
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  images: {
+    remotePatterns: [
+      {
+        hostname: process.env.BIGCOMMERCE_CDN_HOSTNAME ?? '*.bigcommerce.com',
+      },
+    ],
+  },
+  transpilePackages: ['@bigcommerce/components'],
+  typescript: {
+    ignoreBuildErrors: !!process.env.CI,
+  },
+  eslint: {
+    ignoreDuringBuilds: !!process.env.CI,
+    dirs: ['app', 'client', 'components', 'lib', 'middlewares', 'pages', 'providers'],
+  },
+  trailingSlash: true,
+  skipTrailingSlashRedirect: true,
```

Comment:
This is another notable change. This along with removing a line in the middleware fixes the CORs issue.